frontend/src/variants/components/FilterForm/PrioritizationPane.vue

```diff
@@ -38,11 +51,25 @@ const prioEnabledWarning = () => {
 const v$ = useVuelidate()
 
 defineExpose({ v$ })
+
+import { ref } from 'vue'
```

Comment:
An import should happen in the header. Did you run `pretty` and `lint`?

Reply:
Corrected.